Formula/git-recent.rb

```diff
@@ -0,0 +1,16 @@
+class GitRecent < Formula
+  desc "See your latest local git branches, formatted real fancy"
+  homepage "https://github.com/paulirish/git-recent"
+  url "https://raw.githubusercontent.com/paulirish/git-recent/029cc001030d224c961da95502ecee0b54af1989/git-recent"
```

Comment:
Thanks for opening an issue upstream. We can leave this open until a tag/release gets added.